App/ResourceListViewController.swift

```diff
@@ -156,7 +229,19 @@ class ResourceListViewController: UITableViewController {
 				cell.detailTextLabel?.text = "Performed: Unknown Date"
 			}
 		}
-			
+            
+        else if let referralRequestInstance = resource as? ReferralRequest {
+            if let displayName: String = referralRequestInstance.reason?.displayString() {
+                cell.textLabel?.text = displayName
+            } else {
+                cell.textLabel?.text = "Referral Request"
+            }
+            
+            if let specialty: String = referralRequestInstance.specialty?.displayString() {
+                cell.detailTextLabel?.text = "For specialty: " + specialty
```

Comment:
I prefer using `"For specialty: \(specialty)"` but that's just personal preference. :)